src/util/rebind.js

```diff
@@ -24,4 +24,30 @@
         return target;
     };
 
+    function capitalizeFirstLetter(str) {
+        return str[0].toUpperCase() + str.slice(1);
+    }
+
+    /**
+     * Rebinds all the methods from the source component, adding the given prefix. An
+     * optional exclusions parameter can be used to specify methods which should not
+     * be rebound.
+     */
+    fc.util.rebindAll = function(target, source, prefix, exclusions) {
+        exclusions = exclusions || [];
+
+        exclusions.forEach(function(property) {
+            if (!source.hasOwnProperty(property)) {
+                throw new Error('The method ' + property + ' does not exist on the source object');
+            }
+        });
+
+        var bindings = {};
+        for (var property in source) {
+            if (source.hasOwnProperty(property) && exclusions.indexOf(property) === -1) {
+                bindings[prefix + capitalizeFirstLetter(property)] = property;
```

Comment:
I don't think that will work, these bindings need to be 'aliased' to a different property name.